django-ory-cloud/playwright.config.ts

```diff
@@ -0,0 +1,109 @@
+import type { PlaywrightTestConfig } from '@playwright/test'
+import { devices } from '@playwright/test'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  /* Maximum time one test can run for. */
+  timeout: 30_000,
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 1_000
+  },
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* the number of workers, undefined for parallel tests */
+  workers: 1,
```

Comment:
Ideally we keep parallelism enabled and only disable it if there are unsolvable problems. Parallelism adds many desirable features like keeping the test suite as fast as possible, taking advantage of speed improvements in modern processors (which are mostly around being able to do a lot more things in parallel), and making sure the individual tests are truly independent of each other.

```suggestion
  workers: undefined,
```